ql/src/test/results/clientnegative/authorization_uri_export.q.out

```diff
@@ -9,3 +9,4 @@ POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@export_auth_uri
 #### A masked pattern was here ####
+FAILED: SemanticException [Error 10320]: Error while performing IO operation  Exception while writing out the local file
```

Comment:
because we are throwing exception in the analysis phase now and not the execution phase for export.